doc/tutorials/app/orbbec_uvc.markdown

```diff
@@ -0,0 +1,121 @@
+
+- **Process RGB Image**:
+  Using `orbbec_cap.retrieve(None, cv.CAP_OBSENSOR_BGR_IMAGE)` to retrieve the RGB image data. If successfully retrieved, the RGB image is displayed in a window using `cv.imshow("BGR", bgr_image)`.
+
+- **Process Depth Image**:
+  Using `orbbec_cap.retrieve(None, cv.CAP_OBSENSOR_DEPTH_MAP)` to retrieve the depth image data. If successfully retrieved, the depth image is first normalized to a range of 0 to 255, then a false color image is applied, and the result is displayed in a window using `cv.imshow("DEPTH", color_depth_map)`.
+
+- **Keyboard Interrupt**:
+  Using `cv.pollKey()` to detect keyboard events. If a key is pressed, the loop breaks and the program ends.
+
+- **Release Resources**:
+  After exiting the loop, the camera resources are released using `orbbec_cap.release()`.
+#### C++
+
+- **Open Orbbec Depth Sensor**:
+  Using `VideoCapture obsensorCapture(0, CAP_OBSENSOR)` to attempt to open the first Orbbec depth sensor device. If the camera fails to open, an error message is displayed, and the program exits.
+
+- **Retrieve Camera Intrinsic Parameters**:
+  Using `obsensorCapture.get()` to retrieve the intrinsic parameters of the camera, including focal lengths (`fx`, `fy`) and principal points (`cx`, `cy`).
+
+- **Loop to Grab and Process Data**:
+  In an infinite loop, the code continuously grabs data from the camera. The `obsensorCapture.grab()` method attempts to grab a frame.
+
+- **Process RGB Image**:
+  Using `obsensorCapture.retrieve(image, CAP_OBSENSOR_BGR_IMAGE)` to retrieve the RGB image data. If successfully retrieved, the RGB image is displayed in a window using `imshow("RGB", image)`.
+
+- **Process Depth Image**:
+  Using `obsensorCapture.retrieve(depthMap, CAP_OBSENSOR_DEPTH_MAP)` to retrieve the depth image data. If successfully retrieved, the depth image is normalized and a false color image is applied, then the result is displayed in a window using `imshow("DEPTH", adjDepthMap)`. The retrieved depth values are in millimeters and are truncated to a range between 300 and 5000 (millimeter).
+  This fixed range can be interpreted as a truncation based on the depth camera's measurement limits, potentially enhancing the visualization's relevance to a specific depth range.
+
+- **Overlay Depth Map on RGB Image**:
+  Convert the depth map to an 8-bit image, resize it to match the RGB image size, and overlay it on the RGB image with a specified transparency (`alpha`). The overlaid image is displayed in a window using `imshow("DepthToColor", image)`.
+
+- **Keyboard Interrupt**:
+  Using `pollKey()` to detect keyboard events. If a key is pressed, the loop breaks and the program ends.
+
+- **Release Resources**:
+  After exiting the loop, the camera resources are released.
+### Results
+
+#### Python
+
+![RGB And DEPTH frame](images/orbbec_uvc_python.jpg)
+
+#### C++
+
+![RGB And DEPTH And DepthToColor frame](images/orbbec_uvc_cpp.jpg)
```

Comment:
Can you add toggle to show the two pictures, same as the switch to show the two code version?